# The target pose for a single leg
string leg_name
geometry_msgs/Pose pose
